app/experimenter/experiments/tests/test_forms.py

```diff
@@ -256,70 +255,97 @@ def test_empty_slug_raises_error(self):
         self.assertFalse(form.is_valid())
 
 
-class TestExperimentVariantsFormSet(TestCase):
+@parameterized_class(
+    ["form_class"],
+    [[ExperimentVariantsAddonForm], [ExperimentVariantsPrefForm]],
+)
+class TestExperimentVariantsFormSet(MockRequestMixin, TestCase):
```

Comment:
This name isn't great anymore.

Reply:
I still think it applies because that's the name of the class that implements the logic that it's testing.